src/current/v25.2/create-sequence.md

```diff
@@ -296,6 +297,29 @@ For example, to cache 10 sequence values in memory:
 (1 row)
 ~~~
 
+### Cache sequence values per node
+
+ For improved performance, use the `PER NODE CACHE` keyword to cache sequence values in memory at the node level.
```

Comment:
one thing to note -- we will soon be resolving https://github.com/cockroachdb/cockroach/issues/131531, which will make the "per node" behavior be the default behavior when running `CREATE SEQUENCE ... CACHE ...`. we will also introduce a new syntax for the old behavior, like `CREATE SEQUENCE ... PER SESSION CACHE ...`.

this issue should be resolved within 1-2 weeks. shall we wait for that? (or perhaps even just update this docs PR now with the knowledge that the issue will be addressed?)

Reply:
i'd prefer to update the docs after that work goes in using our normal process, i know it might take longer but ... Murphy's Law :-)